spec/models/organisation_spec.rb

```diff
@@ -77,11 +77,11 @@
       end
     end
 
-    context "when an organisation is not deliberately flagged as BEIS" do
-      it "should default to false" do
-        new_organisation = create(:delivery_partner_organisation)
+    context "when an organisation a matched effort provider" do
```

Comment:
Typo: "when an organisation is a matched effort provider"

and a sneaky space on line 81.